api-specification.yml

```diff
@@ -0,0 +1,5516 @@
+openapi: '3.0.3'
+info:
+  title: Mastodon API Specification (https://github.com/mastodon/mastodon)
+  version: '1.0'
```

Comment:
is this supposed to be version for the API? seems weird in any case to call it "1.0"